### PR TITLE
MB-1386: Update plugin to latest SDK versions

### PR DESCRIPTION
**Pull request by rlepinski**
Updates to the latest UA SDKs.

### CODE REVIEW

**Comment by hcrowell** [comment]
:tangerine:

File: README.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ Third Party Packages
 --------------------
  - mod_pbxproj.py - Apache License, Copyright 2012 Calvin Rien
 
+Requirements
+------------
+ - Android SDK installed with the latest updates
```
Comment (marc-scig) [inline README.md]: Android or iOS SDK?

File: README.md [hunk 2]
```diff
@@ -12,6 +12,11 @@ Third Party Packages
...
+ - Unity 5
+
 Setup
 -----
```
Comment (marc-scig) [inline README.md]: iOS setup instructions?

Comment (rlepinski) [inline README.md]: Same for both iOS and Android

File: README.md (hunk 1 above)
Comment (rlepinski) [inline README.md]: You do not need iOS SDK (Xcode) for the plugin, to build the consuming unity project you do. However to build the plugin properly for Android you need the Android SDK installed with the latest support libraries and Google Play Services.

**Pull request merged**